### PR TITLE
EOS-20215: Captive Node CLI Command to generate Node Manifest

### PR DESCRIPTION
**Pull request by tanujashinde0405**
Signed-off-by: Tanuja Shinde <tanuja.shinde@seagate.com>

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

File: node_cli/schema/resource.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
+{
+    "name": "resource",
+    "description": "health and manifest commands",
+    "sub_commands": [
+        {
+            "name": "show",
+            "description": "reshource show health and manifest command",
```
Comment (nitin-seagate) [inline node_cli/schema/resource.json:7]: Change description:
`Show resource health and manifest`

File: node_cli/schema/resource.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            "permissions_tag": "bypass",
+            "args": [
+            {
+                "flag": "--health",
+                "action": "store_true",
+                "help": "Health check for all the resources in the system"
```
Comment (nitin-seagate) [inline node_cli/schema/resource.json:13]: Change description:
`Show resource health`

File: node_cli/schema/resource.json [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+            },
+            {
+                "flag": "--manifest",
+                "action": "store_true",
+                "help": "discover HW/SW Manifest for all resources"
```
Comment (nitin-seagate) [inline node_cli/schema/resource.json:18]: Change description:
`Show resource manifest`

File: node_cli/schema/resource.json [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+            },
+            {
+                "flag": "--resource_type",
+                "type": "str",
+                "help": "Resource type for which resource map is to be fetched e.g 'node>compute[0]>hw>disks' or 'node>compute[0]' or 'node>storage[0]>hw>controllers'"
```
Comment (nitin-seagate) [inline node_cli/schema/resource.json:23]: Change description:
`Filter resources based on type E.g. 'node>compute[0]>hw>disks' or 'node>compute[0]' or 'node>storage[0]>hw>controllers'`

File: node_cli/schema/resource.json [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+            },
+            {
+                "flag": "--resource_state",
+                "type": "str",
+                "help": "The current state of the resources  like 'OK', 'FAULT', 'DEGRADED', 'NA', 'INACTIVE', 'UNKNOWN'"
```
Comment (nitin-seagate) [inline node_cli/schema/resource.json:28]: Change description:
`Filter resources based on state E.g. 'OK', 'FAULT', 'DEGRADED', 'NA', 'INACTIVE', 'UNKNOWN'`

File: node_cli/schema/resource.json [hunk 6]
```diff
@@ -0,0 +1,57 @@
+{
+    "name": "resource",
+    "description": "health and manifest commands",
```
Comment (nitin-seagate) [inline node_cli/schema/resource.json:3]: Change description:
`Discover health and manifest for resources in the system`

**Review by nitin-seagate — CHANGES_REQUESTED** [review]

File: node_cli/schema/resource.json (hunk 1 above)
Comment (tanujashinde0405) [reply node_cli/schema/resource.json]: Done

File: node_cli/schema/resource.json (hunk 2 above)
Comment (tanujashinde0405) [reply node_cli/schema/resource.json]: Done

File: node_cli/schema/resource.json (hunk 3 above)
Comment (tanujashinde0405) [reply node_cli/schema/resource.json]: Done

File: node_cli/schema/resource.json (hunk 4 above)
Comment (tanujashinde0405) [reply node_cli/schema/resource.json]: Done

File: node_cli/schema/resource.json (hunk 5 above)
Comment (tanujashinde0405) [reply node_cli/schema/resource.json]: Done

**Review by nitin-seagate — APPROVED** [review]
LGTM

**Comment by nitin-seagate** [comment]
Please rebase before merging code

**Pull request merged**